extensions.json

```diff
@@ -180,5 +180,19 @@
     "sha256sum": "d7b831c665111aeab5779bea6f5b8f7d5c3afa6d2200d1f79ba2bafbb799497c",
     "foxe": "https://github.com/kminoda/AutowareFoxgloveConverter/releases/download/v0.0.1/kminoda.AutowareFoxgloveConverter-0.0.1.foxe",
     "keywords": ["ros", "autoware"]
+  },
+  {
+    "id": "nvidia.nvblox_foxglove",
+    "name": "Isaac ROS Nvblox Foxglove",
+    "description": "Message converter to visualize nvblox meshes in foxglove.",
+    "publisher": "NVIDIA",
+    "homepage": "https://github.com/NVIDIA-ISAAC-ROS/isaac_ros_nvblox",
+    "readme": "Add_me_after_Isaac3.0_release",
+    "changelog": "Add_me_after_Isaac3.0_release",
+    "license": "Apache 2.0",
+    "version": "2.1.0",
+    "sha256sum": "90af3124c1a68ab5b9b5d3faa480d1c77c0dd902ac1c7a71dea3f445c2df7939",
+    "foxe": "https://github.com/NVIDIA-ISAAC-ROS/.github/blob/main/resources/foxglove_plugins/nvidia.nvblox_foxglove-2.1.0.foxe",
```

Comment:
This url needs to point to a location that will trigger a file download (see the other entries for example). When I visit this URL I am taken to a github webpage. That won't work in the app since it won't know what to do with that page.

Reply:
I think a link like `https://github.com/NVIDIA-ISAAC-ROS/.github/raw/main/resources/foxglove_plugins/nvidia.nvblox_foxglove-2.1.0.foxe` will work.

Reply:
Thanks for pointing this out. I've fixed the URL